mmdet/apis/train.py

```diff
@@ -68,6 +68,51 @@ def set_random_seed(seed, deterministic=False):
         torch.backends.cudnn.benchmark = False
 
 
+def scale_lr(cfg, logger):
+    """Automatically scaling LR according to GPU number and sample per GPU.
+
+    Args:
+        cfg (config): training config.
+        logger (logging.Logger): logger.
+    """
+    enable_auto_scale_lr = cfg.get('enable_auto_scale_lr', False)
+    if enable_auto_scale_lr is False:
+        logger.info('You disabled automatically scaling LR.')
```

Comment:
You disabled automatically scaling LR -> Automatic scaling of learning rate (LR) has been disabled.

Reply:
Done

Reply:
Done